repository.json

```diff
@@ -94,6 +94,22 @@
 				}
 			]
 		},
+		{
+			"details": "https://github.com/sublimelsp/LSP-astro",
+			"name": "LSP-astro",
+			"labels": [
+				"auto-complete",
+				"linting",
+				"code navigation",
+				"intellisense"
+			],
+			"releases": [
+				{
+					"sublime_text": ">=4070",
```

Comment:
Unless you expect this to have some ST4-only logic later, it doesn't need to be limited to ST4 as lsp_utils is backwards-compatible.

Reply:
I don't want to struggle with any ST3 related request. Especially as LSP isn't developed against ST3 anymore as well, AFAIK.